source/features/highest-rated-comment.tsx

```diff
@@ -0,0 +1,63 @@
+import './highest-rated-comment.css';
+import React from 'dom-chef';
+import select from 'select-dom';
+import features from '../libs/features';
+import * as icons from '../libs/icons';
+
+function getCount(reaction: HTMLElement): number {
+	return Number(reaction.textContent!.match(/\d+/)![0]);
+}
+
+function init(): false | void {
+	let highest;
+	const likes = select.all('.js-discussion .js-timeline-item:nth-child(n+6) [aria-label*="reacted with thumbs up"]');
+	for (const like of likes) {
+		const count = getCount(like);
+		const dislike = select('[aria-label*="reacted with thumbs down"]', like.parentElement!);
+
+		if (dislike && getCount(dislike) >= count / 2) {
+			continue; // Controversial comment
+		}
+
+		if (!highest || count > highest.count) {
+			highest = {like, count};
+		}
+	}
+
+	if (!highest || highest.count < 10) {
+		return false;
+	}
+
+	const event = highest.like.closest('.js-timeline-item')!;
+	const text = select('.comment-body', event)!.textContent!.substring(0, 100);
+	const avatar = select('.timeline-comment-avatar', event)!.cloneNode(true);
+	const {hash} = select<HTMLAnchorElement>('.timestamp', event)!;
+
+	select('.unminimized-comment', event)!.classList.add('rgh-highest-rated-comment');
+	event.parentElement!.firstElementChild!.after((
```

Comment:
The highest-rate comment might still be within those first 5 comments, so we should still highlight that one _but_ we should not add the link.

Example: https://github.com/facebook/react/issues/12014

To do, perhaps you can use:
```suggestion
	const position = select.all('.js-comment').indexOf(select('.js-comment', event));
	if (position > 5) {
		event.parentElement!.firstElementChild!.after((
```

Reply:
Used >=4 to hit from the 5th, hope that's fine.